src/components/settings/data/hooks.js

```diff
@@ -60,7 +61,59 @@ export const useLinkManagement = (enterpriseUUID) => {
   };
 };
 
+const initialCustomerAgreementState = {
+  netDaysUntilExpiration: 0,
+};
+/*
+ * @param {Objet {enterpriseId: string}}
+ * @returns {customerAgreement: Object, loadingCustomerAgreement: bool}
+ * customerAgreement:{
+ *  netDaysUntilExpiration: number
+ * }
+ */
+export const useCustomerAgreementData = ({ enterpriseId }) => {
+  const [customerAgreement, setCustomerAgreement] = useState(initialCustomerAgreementState);
+  const [loadingCustomerAgreement, setLoadingCustomerAgreement] = useState(true);
```

Comment:
Assuming `loadCustomerAgreementData` is going to set `loadingCustomerAgreement` to true below, we can probably have this initial state be defaulted to false?